spec/parser.js

```diff
@@ -215,6 +222,14 @@ describe('parser', function() {
     shouldThrow(function() {
       astFor('{{{{goodbyes}}}} {{{{/hellos}}}}');
     }, Error, /goodbyes doesn't match hellos/);
+
+    shouldThrow(function() {
+      astFor('{{foo ...}}');
+    }, Error, /Parse error on line 1/);
+
+    shouldThrow(function() {
+      astFor('{{foo ...=lol ...baz}}');
+    }, Error, /Parse error on line 1/);
```

Comment:
Yes, I'll add a test. Basically this PR doesn't change the rule that params are on the left, hashy things on the right.